assets/section-main-product.css

```
@@ -374,7 +374,8 @@ a.product__text {
 .product--no-media .product__title,
 .product--no-media .product__text,
 .product--no-media noscript .product-form__input,
-.product--no-media .product__tax  {
+.product--no-media .product__tax,
+.product--no-media shopify-payment-terms   {
```

This extra spacing was already here. Sorry about that. Would you mind fixing it for us? 
```suggestion
.product--no-media shopify-payment-terms {
```

---

Gah! yes, thank you, sorry about that.